docs/modules/ROOT/pages/integrations/database.adoc

```diff
@@ -0,0 +1,114 @@
+= Database Integrations
+
+== Overview
+
+KubeArchive requires a database to store resources.
+This document list the databases supported, how to set up the database schema
+and how to add new database integrations.
+
+=== PostgreSQL
+
+The only currently full supported.
+KubeArchive integration test suite runs with PostgreSQL 16.6
+
+The PostgreSQL implementation is available
+link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/postgresql.go[here].
+
+=== MariaDB
+
+This integration is still in progress.
+
+The MariaDB implementation is available
+link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/mariadb.go[here].
+
+== Configuration and Customization
+
+KubeArchive offers a [filename]`kubearchive.sql` file with the DDL to create
+the schema as part of the installation.
+
+[IMPORTANT]
+====
+The provided schema includes indexes.
+
+KubeArchive users should review and customize the indexes
+if needed based on the expected queries and the amount and
+the distribution of the expected archived data.
+====
+
+Specific instructions for the configuration of the PostgreSQL Database are available in
+xref:/getting-started/instllation.adoc#_install_and_configure_kubearchive[Install and Configure KubeArchive].
+
+== Add new database integrations
+
+To add a new database integration follow the instructions:
+
+1. Create a new file under `pkg/database` with the name of the new database type.
+2. Include an
+link:https://go.dev/doc/effective_go#init[init function]
+that:
+
+* Inserts in the
+link:https://github.com/maruiz93/kubearchive/blob/postgresql-version-update/pkg/database/database.go#L27[`RegisteredDBCreators`]
```

Comment:
Should this point to our main repository instead?

```suggestion
link:https://github.com/kubearchive/kubearchive/blob/postgresql-version-update/pkg/database/database.go#L27[`RegisteredDBCreators`]
```

Reply:
ops, yes, it should

Reply:
fixed